### ISSUE TITLE
buildserver: cleanup task for workspace on nodes

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#3054]
It seems like currently the workspace is accumulating without ever being cleaned up.

On v2 /home/jenkins/workspace contains 148 directories which need 44G space.

Hopefully there is a ready-made Jenkins plugin for this.

**Comment by markus2330** [comment]
@mpranj is this task already done?

**Comment by mpranj** [comment]
> is this task already done?

I don’t know. I have not been working on this. @Mistreated do you know what the status is here?

**Comment by markus2330** [comment]
Let me rephrase: is this task still relevant as we found out that docker was the main cause of making the discs full.

If it is not enough, the next step would be to change how many jobs are kept by Jenkins, as this is the smallest effort (writing our own cleaner cron job would be much more difficult).

**Comment by mpranj** [comment]
No I think we did not have any problems with disk space recently. (I sometimes prune docker stuff manually when I have time and update the build agents)

Let’s close this for now as the problems did not occur any more.
